src/components/Clothing.js

```diff
@@ -1,11 +1,69 @@
-import styled from 'styled-components/macro'
+import { useState } from 'react'
+import styled, { css } from 'styled-components/macro'
 
-function Clothing({ imageUrl }) {
-  return <StyledImage src={imageUrl} alt="" />
+function Clothing({ titel, color, pattern, material, fitting, imageUrl }) {
+  const [showDetails, setShowDetails] = useState(false)
+
+  return (
+    <StyledSection>
+      <StyledButton
+        showDetails={showDetails}
+        onClick={() => {
+          if (showDetails) {
+            setShowDetails(false)
+          } else {
+            setShowDetails(true)
+          }
+        }}
+      >
+        <StyledImage src={imageUrl} alt="" />
+        {showDetails ? (
+          <StyledWrapper>
+            <h2>{titel}</h2>
+            <p>Farbe: {color}</p>
+            <p>Muster: {pattern}</p>
+            <p>Material: {material}</p>
+            <p>Passform: {fitting}</p>
+          </StyledWrapper>
+        ) : (
+          <div></div>
+        )}
```

Comment:
I think it would be a bit nicer to read with the logical AND operator, you also then don't have an empty `div` in your markup.
```suggestion
        {showDetails && (
          <StyledWrapper>
            <h2>{titel}</h2>
            <p>Farbe: {color}</p>
            <p>Muster: {pattern}</p>
            <p>Material: {material}</p>
            <p>Passform: {fitting}</p>
          </StyledWrapper>
        )}
```